migration-guides/ESLINT_FLAT_CONFIG.md

```diff
@@ -0,0 +1,104 @@
+# Migration to ESLint Flat Configuration
+
+[ESLint RC format](https://eslint.org/docs/latest/use/configure/configuration-files-deprecated) is deprecated in favor of [ESLint FlatConfig](https://eslint.org/blog/2022/08/new-config-system-part-2/).
+To migrate, you can follow their [migration guide](https://eslint.org/docs/latest/use/configure/migration-guide).
+
+On Otter side, an ESLint Flat Configuration is exposed by the [`@o3r/eslint-config`](https://www.npmjs.com/package/@o3r/eslint-config) package.
+
+## Rules changes
+
+> [!NOTE]
+> All the rules set to `warn` has been upgraded to `error`.
+
+Below you can fing, the list of differences by plugin between configurations exposed by `@o3r/eslint-config-otter` and `@o3r/eslint-config`.
+
+### Angular ESLint rules
+
+#### Added
+
+We added the [recommended rules](https://github.com/angular-eslint/angular-eslint/blob/main/packages/angular-eslint/src/configs/README.md#angular-eslinttemplate-accessibility) for accessibility in the template.
```

Comment:
```suggestion
We added the [template accessibility recommended rules](https://github.com/angular-eslint/angular-eslint/blob/main/packages/angular-eslint/src/configs/README.md#angular-eslinttemplate-accessibility).
```

Reply:
@sdo-1A : What do you think for this one, I am not convince by my suggestion ?

Reply:
I would make a slight change: `recommended rules for template accessibility`
what do you think?